### PR TITLE
feat: implement tx summary counter

### PR DESCRIPTION
**Pull request by akhilkumarpilli**
Closes: #29 

Load test transactions summary will be shown as below:

```
Testing Stats:
-----------------------------
Number of transactions executed: 50
Number of messages executed: 50
Number of successful transactions: 1 (2.0%)
Number of failed transactions: 49 (98.0%)
        
Failures:
Failed with code 11 (out of gas): 25
Failed with code 32 (incorrect account sequence): 24
-----------------------------
```

### CODE REVIEW

**Comment by anilCSE** [comment]
> Load test transactions summary will be shown as below:
> 
> ```
> Testing Stats:
> -----------------------------
> Number of transactions executed: 50
> Number of messages executed: 50
> Number of successful transactions: 1 (2.0%)
> Number of failed transactions: 49 (98.0%)
>         
> Failures:
> Failed with code 11 (out of gas): 25
> Failed with code 32 (incorrect account sequence): 24
> -----------------------------
> ```

Great. So does it show other errors too, something like

```
Failed with code 3:
Errors: (show couple of errors here as they might be related to some bugs)
-
```

File: python-qa-tools/stats/stats.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import json, logging
+from db import db
+
+logging.basicConfig(format="%(message)s", level=logging.DEBUG)
+
+COL_NAME = "test_stats"
+TX_TYPE = "tx"
+QUERY_TYPE = "query"
+
```
Comment (anilCSE) [inline python-qa-tools/stats/stats.py:9]:
```suggestion
```

File: python-qa-tools/stats/stats.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+def insert_stat(data):
+    _ = db[COL_NAME].insert_one(data)
+
+
+def check_status(test_type, cmd_type, output, err):
```
Comment (anilCSE) [inline python-qa-tools/stats/stats.py:15]: can we rename it to better naming. How about `track` or `record` or maybe `record_stat`?

File: python-qa-tools/stats/stats.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+logging.basicConfig(format="%(message)s", level=logging.DEBUG)
+
+COL_NAME = "test_stats"
+TX_TYPE = "tx"
+QUERY_TYPE = "query"
+
+
+def insert_stat(data):
+    _ = db[COL_NAME].insert_one(data)
+
```
Comment (anilCSE) [inline python-qa-tools/stats/stats.py:13]:
```suggestion
```

File: python-qa-tools/utils.py [hunk 4]
```diff
@@ -1,61 +1,81 @@
...
+        )
+
 
 # The utility function `exec_command` is used to execute the cosmos-sdk based commands.
-def exec_command(command):
-    stdout, stderr = subprocess.Popen(command.split(),
-                                    stdout = subprocess.PIPE,
-                                    stderr = subprocess.PIPE).communicate()
-    return stdout.strip().decode(), stderr.strip().decode()
+def exec_command(command, test_type=None, cmd_type=None):
```
Comment (anilCSE) [inline python-qa-tools/utils.py:26]: is there a way to keep it clean. may be we can check for `tx` or `query` substring and get the cmd_type? Similarly how about using an ENV for test_type.

**Review by anilCSE — CHANGES_REQUESTED** [review]

File: python-qa-tools/stats/stats.py (hunk 1 above)
Comment (akhilkumarpilli) [reply python-qa-tools/stats/stats.py:9]: This is due to python formatter. Every function in python will be separated from other with two empty lines.

**Review by anilCSE — APPROVED** [review]
thank you :tada:

**Pull request merged**